apps/pl/__tests__/pl.test.js

```diff
@@ -0,0 +1,5 @@
+const BUILD_TARGET = 'pl';
```

Comment:
I've never seen the __tests__ convention for a testing folder. Why need the underscores?

Reply:
Because Jest.